app/components/containers/containersController.js

```diff
@@ -231,6 +243,30 @@ angular.module('containers', [])
     batch($scope.containers, Container.remove, "Removed");
   };
 
+  $scope.confirmRemoveAction = function () {
+    if ($scope.state.selectedItemRunning > 0) {
+      force = true;
+      ModalService.confirm({
+        title: "Are you sure?",
+        message: "Some selected containers are running. Please confirm your action",
```

Comment:
Replace `Some selected containers are running. Please confirm your action` with `You are about to remove one or more running containers.`